include/benchmark/benchmark.h

```diff
@@ -575,7 +575,10 @@ class State {
 
   BENCHMARK_ALWAYS_INLINE
   size_t iterations() const {
-    return (max_iterations - total_iterations_ + batch_leftover_);
+    if (!started_) {
```

Comment:
maybe the option to slightly improve codegen by assuming branch prediction?

Reply:
I'm not sure what you mean. Were you looking for BENCHMARK_BUILTIN_EXPECT or something else?